acvm_js/test/node/execute_circuit.test.ts

```diff
@@ -205,3 +205,31 @@ it("successfully executes two circuits with same backend", async function () {
   expect(solvedWitness0).to.be.deep.eq(expectedWitnessMap);
   expect(solvedWitness1).to.be.deep.eq(expectedWitnessMap);
 });
+
+it.only("successfully executes 500 circuits with same backend", async function () {
```

Comment:
Would be good to keep a test like this for regression purposes without the console.log statements